bedevere/bpo.py

```diff
@@ -83,10 +83,67 @@
         await set_status(event, gh)
 
 
+@router.register("issue_comment", action="edited")
+@router.register("issue_comment", action="created")
```

Comment:
I'm thinking that the hyperlinking should also be done on `pull_request` `opened` and `edited` events.
The body will be `event.data["pull_request"]["body"]` instead of `event.data["comment"]["body"]`.
And the patch url to be used in line 93 will be `event.data["pull_request"]["issue_url"]`.

Reply:
Thanks for reviewing!  :slightly_smiling_face: 
Just a small question:
Wouldn't it be better if the bot hyperlinks `bpo-` text in the comment? As 
* the pull request body already contains the link of the issue. 
* Also, few (510 out of 7,525) PRs have `bpo-` text in the body.
* The issue also references to hyperlinking
Please correct me if I am wrong.

Reply:
Yeah I meant to do this in both `issue_comment` and `pull_request` body.

The PR body already contains the link to the bpo that is mentioned in the PR title, and we should leave that as is.
I just think it would be nice to be also able to refer to other bpo tickets in the PR body.

Reply:
Okay. Thanks!
I get it now :slightly_smiling_face: